pkg/storage/calico/converter.go

```diff
@@ -33,63 +33,63 @@ func aapiError(err error, key string) error {
 // This is common code. Refactor this workflow.
 func convertToAAPI(libcalicoObject runtime.Object) (res runtime.Object) {
```

Comment:
This function might also be obsolete now, since there are no longer two types to convert between!